pkg/odp/lru_cache.go

```diff
@@ -0,0 +1,110 @@
+/****************************************************************************
+ * Copyright 2022, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package odp //
+package odp
+
+import (
+	"container/list"
+	"sync"
+	"time"
+)
+
+// Cache is used for caching ODP segments
+type Cache interface {
+	Save(key string, value interface{})
```

Comment:
can we add specific type of `interface` instead of `interface{}`

Reply:
interface has no further types in go.